setup.py

```diff
@@ -55,8 +56,9 @@ def remove_bdist_rpm_source_file():
     'version': '2.2.6',
     'author': [sdw, jt, ag, kh],
     'maintainer': [sdw, jt, ag, kh],
-    'packages': ['vsc', 'vsc.utils', 'vsc.install'],
+    'packages': ['vsc', 'vsc.install', 'vsc.test', 'vsc.utils'],
```

Comment:
maybe I should move the tests one level deeper even, to `vsc.test.vsc_base`, to avoid clashes with the other `vsc` packages when they move to this scheme as well?